nwbwidgets/controllers/group_and_sort_controllers.py

```diff
@@ -136,6 +136,7 @@ def __init__(
                 "TimeIntervals": "trials",
                 "Units": "units",
                 "PlaneSegmentation": "image_planes",
+                "FibersTable": "traces",
             }
             desc = dt_desc_map[dynamic_table.neurodata_type] if dynamic_table is not None else "traces"
```

Comment:
```suggestion
            desc = dt_desc_map.get(dynamic_table.neurodata_type, "traces") if dynamic_table is not None else traces
```

Reply:
let's add this default in so we don't get an error the next time a table is added